plugins/github.js

```diff
@@ -12,12 +12,13 @@ hoverZoomPlugins.push({
             res.push(img);
         });
 
-        $('img[src*="private-user-images"]').each(function () {
-            let img = $(this);
-            img.data('hoverZoomSrc', [img.attr('src')]);
-            res.push(img);
+        $('img[src*="user-images"]:not(.hoverZoomMouseover)').addClass('hoverZoomMouseover').one('mouseover', function () {
```

Comment:
I wonder why hacks like `addClass('hoverZoomMouseover')` are necessary. Shouldn't this class be always set automatically by `hoverZoom.prepareLink`?

Reply:
'hoverZoomLink' is set by prepareLink(). Should it set hoverZoomLink instead or in addition?

Reply:
I'm not sure I understand the purpose of adding `.hoverZoomMouseover` and using it in a selector. Since you are adding it _after_ selecting, that should guarantee that none of the existing elements would have it. And since this plugin is only executed once, there's no point of adding this class because there noone to check after the plugin is done.

Reply:
Ohhh, I understand what you're saying now. I thought that part was necessary as it didn't seem to work without it, but after troubleshooting, it turns out it was something else. It is removed as you suggested